server/plugin/webhook.go

```diff
@@ -270,6 +272,10 @@ func (p *Plugin) postPullRequestEvent(event *github.PullRequestEvent) {
 		}
 
 		if action == "opened" {
+			policy := bluemonday.StrictPolicy()
+			policy.SkipElementsContent("details")
+			newPRMessage = strings.ReplaceAll(newPRMessage, "<details>", "`(details collapsed)`<details>")
```

Comment:
Could you please post an example on how the output looks like?

Reply:
@hanzei  Please find the screenshot
![Screenshot 2021-08-13 at 12 36 26 AM](https://user-images.githubusercontent.com/86650698/129254407-7745861d-4fc7-404b-80ff-69abeef00f4e.png)

Reply:
I'm 2/5 that it's more confusing if we replace the details with this placeholder. Does it make sense to only output the conentent of  `<summary>` here?

Reply:
@hanzei   Do you mean to remove the details section completely and the details collapsed completely and just show the rest of the part ?

Reply:
I mean removing the body of the details section while keeping the `<summary` header.

Reply:
Does that make things more clear?

Reply:
@hanzei Have made the following changes
attaching the screenshot for same
```
PR created In Github
![Screenshot 2021-08-18 at 11 17 56 PM](https://user-images.githubusercontent.com/86650698/129947404-1113d226-0f7a-4e85-8000-ea9ad5307b20.png)
Trigger Message in MM GIthub bot 
![Screenshot 2021-08-18 at 11 18 11 PM](https://user-images.githubusercontent.com/86650698/129947442-41d85f26-7181-4d20-9843-8489993808e6.png)
```

Reply:
Actually, just seeing the summary does look a bit confusing. 1/5 that it's more clear to the user if we remove the whole `datails` block. What do you think?

Reply:
@hanzei  i agree this looks too ugly ,
we can simply remove the details  block and keep it clean